packages/app/components/settings/privacy-and-security.tsx

```diff
@@ -1,43 +1,62 @@
-import { Platform } from "react-native";
+import { Linking, Platform } from "react-native";
 
+import { Alert } from "@showtime-xyz/universal.alert";
 import { ScrollView } from "@showtime-xyz/universal.scroll-view";
+import { Text } from "@showtime-xyz/universal.text";
 import { View } from "@showtime-xyz/universal.view";
 
 import { useHeaderHeight } from "app/lib/react-navigation/elements";
 
-import { SettingBody } from "./setting-content";
-import { SettingHeaderSection } from "./setting-header";
-import {
-  AccountSettingItem,
-  AccountSettingItemProps,
-} from "./settings-account-slot";
-
-const list = [
-  {
-    id: 1,
-    title: "Blocked Accounts",
-    icon: "lock",
-    subRoute: "blocked-list",
-  },
-];
+import { AccountSettingItem } from "./settings-account-item";
 
 export const PrivacyAndSecuritySettings = () => {
   const headerHeight = useHeaderHeight();
 
-  const renderSetting = (item: AccountSettingItemProps) => {
-    return (
-      <AccountSettingItem
-        key={`privacy-and-security-setting-${item.id}`}
-        {...item}
-      />
-    );
-  };
-
   return (
-    <ScrollView tw="w-full">
+    <ScrollView tw="w-full bg-white dark:bg-black md:bg-transparent">
       {Platform.OS !== "android" && <View style={{ height: headerHeight }} />}
-      <SettingHeaderSection title="Privacy & Security" />
-      <SettingBody>{list.map(renderSetting)}</SettingBody>
+      <View tw="mx-auto mt-4 w-full max-w-screen-lg px-4 md:mt-8">
+        <Text tw="font-space-bold text-2xl font-extrabold text-gray-900 dark:text-white">
+          Privacy & Security
+        </Text>
+        <View tw="h-8" />
+        <View tw="rounded-2xl bg-white px-0 dark:bg-black md:px-4">
+          <AccountSettingItem
+            title="Blocked Accounts"
+            buttonText="View"
+            onPress={() => {
+              Alert.alert("🚧 Coming soon");
```

Comment:
If we don't have that function, we should not even show it. Apple might reject the app if they find this.

Reply:
yeah hide it is the best way right now!